shared/src/androidMain/kotlin/co/touchlab/kampstarter/KoinAndroid.kt

```diff
@@ -21,13 +21,14 @@ actual val platformModule: Module = module {
         )
     }
 
+    single<SharedPreferences> {
```

Comment:
Started doing similar changes in Android to parallel iOS, but didn't go all the way. Would plan to do more here if we move forward with this.